src/components/ScholarshipAmountField.jsx

```diff
@@ -1,28 +1,32 @@
 import React from 'react';
 import {
-  FormLabel,
-  Radio,
-  RadioGroup,
-  FormControl,
-  FormControlLabel,
+  InputLabel,
+  Grid,
+  Select,
+  MenuItem,
   FormHelperText,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
+import RemoveIcon from '@material-ui/icons/Remove';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 import AmountTextField from './AmountTextField';
 
-const useStyles = makeStyles({
-  amountFieldStyle: {
-    display: 'flex',
-    alignItems: 'center',
+const useStyles = makeStyles((theme) => ({
+  amountSelect: {
+    minWidth: 150,
   },
-});
+  dash: {
+    marginTop: theme.spacing(2),
```

Comment:
This is making some assumptions about the size of the text. Instead of this I suggest making this vertically centered:
```suggestion
    height: '100%',
    textAlign: 'center',
```

Got this from [SO](https://stackoverflow.com/a/8865463) and testing it out on the web page inspector for validation